src/app/options.cpp

```diff
@@ -235,6 +240,69 @@ namespace
         return o == s;
     }
 
+    // Option that is explicitly set to true or false, and whose value is undefined when unspecified.
+    // May not have a shortcut.
+    class TriStateBoolOption: protected StringOption
+    {
+    public:
+        constexpr TriStateBoolOption(const char *name)
+            : StringOption {name}
+        {
+        }
+
+        using StringOption::operator==;
+        
+        QString usage() const
+        {
+            return StringOption::usage(QLatin1String("true|false"));
+        }
+
+        TriStateBool value(const QString &arg) const
+        {
+            QString val = StringOption::value(arg);
+            
+            if (val.toUpper() == QLatin1String("TRUE") || val == QLatin1String("1")) {
+                return TriStateBool::True;
+            }
+            else if (val.toUpper() == QLatin1String("FALSE") || val == QLatin1String("0")) {
+                return TriStateBool::False;
+            }
+            else {
+                //: Translation must use English "true" and "false"
```

Comment:
Will this go into Transifex? Why don't you use the second argument of `QObject::tr()`?